executor/replace.go

```diff
@@ -202,6 +207,7 @@ func (e *ReplaceExec) exec(ctx context.Context, newRows [][]types.Datum) error {
 			return err
 		}
 	}
+	memTracker.Consume(int64(txn.Size()))
```

Comment:
this also have potential be duplicate add consume amount if user use batch insert(although we are not recommend use batch insert)

`exec` be executed for each batch, so second batch will see first batch's memory in stmt buffer